locations/spiders/ubb_bg.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import re
+
+import scrapy
+from scrapy import Spider
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.hours import DAYS_BG, OpeningHours, day_range, sanitise_day
+from locations.items import Feature
+from locations.user_agents import FIREFOX_LATEST
+
+
+class UbbBGSpider(Spider):
+    name = "ubb_bg"
+    item_attributes = {"brand": "Обединена българска банка", "brand_wikidata": "Q7887555"}
+    start_urls = ["https://www.ubb.bg/offices/pins"]
+    user_agent = FIREFOX_LATEST
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+        "DEFAULT_REQUEST_HEADERS": {"Accept": "application/json, text/javascript, */*"},
+    }
```

Comment:
```suggestion
    }
    requires_proxy = True
```

Reply:
Thank you! It works as expected now. I assume it also needed the cookie, which is why it didn't work with the proxy the first time.